packages/mdc-ripple/util.js

```diff
@@ -15,13 +15,21 @@
  */
 
 /** @private {boolean|undefined} */
+let supportsCssVariables_;
+
+/** @private {boolean|undefined} */
 let supportsPassive_;
 
 /**
  * @param {!Window} windowObj
+ * @param {boolean=} forceRefresh
```

Comment:
The unit tests all run in a single environment and need to re-test that function several times with different window object mocks to exercise the different code paths, so they need a way to force it to re-evaluate even if there's a cached result.  Meanwhile for typical use the cached result is correct because the page is always in the same browser, and it doesn't make sense to re-evaluate every time you instantiate a ripple (there could be a lot on a page).